backend/app/controllers/users.rb

```diff
@@ -41,12 +42,16 @@ class ArchivesSpaceService < Sinatra::Base
 
   Endpoint.get('/users')
     .description("Get a list of users")
-    .params()
+    .params([ "show_inactive", BooleanParam, "Flag to show inactive users", :optional => true ])
     .paginated(true)
     .permissions([])
     .returns([200, "[(:resource)]"]) \
   do
-    handle_listing(User, params, {:exclude => {:id => User.unlisted_user_ids}})
+    if params[:show_inactive]
+      handle_listing(User, params, {:exclude => {:id => User.unlisted_user_ids}})
+    else
+      handle_listing(User, params, {:exclude => {:id => User.unlisted_user_ids, :is_active_user => 0}})
```

Comment:
```suggestion
      handle_listing(User, params, {:exclude => {:id => User.unlisted_user_ids}, :is_active_user => 1})
```
I think sending two keys to exclude will only exclude records matching both conditions, so you have to stick to one exclude field (or modify crud_helpers to support exclude chaining) and use a filter for `:is_active_user`